rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -1481,6 +1481,13 @@ stream {
             proxy_next_upstream_timeout             {{ $location.Proxy.NextUpstreamTimeout }};
             proxy_next_upstream_tries               {{ $location.Proxy.NextUpstreamTries }};
 
+            {{ if or (eq $location.BackendProtocol "GRPC") (eq $location.BackendProtocol "GRPCS") }}
+            # Grpc settings
```

Comment:
question: what happens if the user does not set the proxy configuration timeout? I can't remember if there are defaults for connecttimeout, sendtimeout, etc.

Reply:
The proxy default timeouts (5s,60s,60s) is configured in defuault backend in

https://github.com/Anddd7/ingress-nginx/blob/5badb96d909242e02eb3f0d95372c2a298ec5325/internal/ingress/controller/config/config.go#L863-L867